### PR TITLE
Check for the presence of tests in more PRs

### LINKED ISSUE #35: Check for the presence of tests in more PRs
We currently warn about layout changes that don't include tests. I think we should extend this to every PR that touches components/script, components/gfx, components/style, and components/net, since those are the most likely to contain testable changes.

Code: `handlers/missing_test/__init__.py`

### PR DESCRIPTION
**Pull request by PythonNut**
Fixes #35.

I've done my best to maximize readability. If you have any complaint, I'd be happy to fix it.

### CODE REVIEW

File: handlers/missing_test/__init__.py [hunk 1]
```diff
@@ -1,20 +1,36 @@
 from eventhandler import EventHandler
 
-reftest_required_msg = 'These commits modify layout code, but no reftests are modified. Please consider adding a reftest!'
+REFTEST_REQUIRED_MSG = 'These commits modify {0} code, but no reftests are modified. Please consider adding a test!'
```
Comment (jdm) [inline handlers/missing_test/__init__.py]: Let's call this "TEST_REQUIRED_MSG", and change the remaining "reftests" to "tests".

**Comment by jdm** [comment]
This looks great! Could we also extend the tests for this handler by checking that the wpt and unit directories suppress the warning?

**Comment by jdm** [comment]
We could steal the new tests from #38 for that purpose.

File: handlers/missing_test/__init__.py (hunk 1 above)
Comment (PythonNut) [inline handlers/missing_test/__init__.py]: Sure, sorry. I missed that for some reason.

**Comment by PythonNut** [comment]
Is there any way we can still give @yoongkang authorship credit for the tests?

**Comment by KiChjang** [comment]
Yes, cherry-pick his second commit on his PR (i.e. the one with the SHA of `82128e6076bd89d5ea78607435cf506965f45aa0`).

**Comment by PythonNut** [comment]
@KiChjang thanks. I also had to [modify the commit](http://stackoverflow.com/questions/5717026/how-to-git-cherry-pick-only-changes-to-certain-files) and [manually set the author](http://stackoverflow.com/questions/3042437/change-commit-author-at-one-specific-commit).

**Comment by jdm** [comment]
Lovely! Thanks!

**Pull request merged**